engine/src/org/pentaho/di/job/entries/zipfile/JobEntryZipFile.java

```diff
@@ -977,9 +970,15 @@ public Result execute( Result previousResult, int nr ) {
     return result;
   }
 
+  private Result errorResult( Result result ) {
+    result.setNrErrors( 1 );
```

Comment:
@mbatchelor, that makes sense, but, in fact, I only extracted old code into a separate method to reduce duplication. Changing errors' counting should be done separately